### DISCUSSION TITLE
Installing brew hangs

### DISCUSSION THREAD

**Discussion by lgprall** [Homebrew/discussions#5750] [Getting started]
The install hangs on:

Press RETURN/ENTER to continue or any other key to abort:
==> /usr/bin/sudo /usr/sbin/chown -R larry:admin /opt/homebrew

I can run the command manually and it succeeds.

The reason I am trying to (re-)install brew is that it stopped working after a restore. All functional commands hang. I can do a "brew list" or a "brew help", but doing a "brew update" or a "brew upgrade" or "brew doctor" or any other command hangs. This is on an iMac M1, running 15.1.1 and freshly restored from a TimeMachine backup. Everything was working before the upgrade/restore.

Any ideas??

**Comment by gromgit** [comment]
Try `brew update-reset`, then `brew doctor`.

**Comment by lgprall** [comment]
That worked! Thanks.Sent from my eyePad, of course...On Nov 20, 2024, at 9:52 PM, Adrian Ho ***@***.***> wrote:﻿
Try brew update-reset, then brew doctor.
